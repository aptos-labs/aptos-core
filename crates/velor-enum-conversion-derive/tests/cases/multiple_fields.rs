// Copyright Â© Velor Foundation
// SPDX-License-Identifier: Apache-2.0

use velor_enum_conversion_derive::EnumConversion;

#[derive(EnumConversion)]
enum Messages {
    Test(String, String)
}

fn main() {

}
