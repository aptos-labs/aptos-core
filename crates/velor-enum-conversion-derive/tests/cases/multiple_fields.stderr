error: enum variant must have exactly one field
 --> tests/cases/multiple_fields.rs:7:1
  |
7 | / enum Messages {
8 | |     Test(String, String)
9 | | }
  | |_^
